moveit_core/constraint_samplers/test/pr2_arm_kinematics_plugin.cpp

```diff
@@ -167,7 +164,7 @@ int PR2ArmIKSolver::cartToJntSearch(const KDL::JntArray& q_in, const KDL::Frame&
   KDL::JntArray q_init = q_in;
   double initial_guess = q_init(free_angle_);
 
-  ros::Time start_time = ros::Time::now();
+  rclcpp::Time start_time = rclcpp::Clock().now();
```

Comment:
We want RCL_ROS_TIME here not RCL_SYSTEM_TIME

http://docs.ros2.org/latest/api/rcl/time_8h.html#a5c734f508ce06aec7974af10ad09d071a423b9ea49c5f5119c3b9b812c01e6bc9

Reply:
https://github.com/ros-planning/moveit2/pull/60/commits/01ca1b5d2768669845acffa9324e26fa89d13752